(maint) add change length PR check

er0ck opened the pull request:
* add a rake task to check LoC in diff against the target branch, or master
* add the check to the travis checks

Commit:
(maint) add change length PR check

* add a rake task to check LoC in diff against the target branch, or master
* add the check to the travis checks

billclaytor commented:
The changes to `Rakefile` and `.travis.yml` look good, the checks pass, and `diff_length` is referenced in the build log. Since this is designed to be run in the context of a PR it would seem to require a PR that exceeds `max_length` to test that condition. Would it be worth reducing the value temporarily to test it?

er0ck commented:
changed the max_length to 1 to test it out :-D

[edit] ... and, it didn't work.  wtf
[edit2] okay, i think i fixed it.  it won't work if we want to cascade this task with another because i put an exit in.  but it works for here and the way travis is setup

yep.  that worked: https://travis-ci.org/puppetlabs/doctor_teeth/jobs/351080859

i'll change the value back now

billclaytor commented:
Looks good; glad the negative test found an issue.
